yt_dlp/extractor/playsuisse.py

```diff
@@ -134,12 +135,35 @@ class PlaySuisseIE(InfoExtractor):
             id
             url
         }'''
+    _ID_TOKEN = None
+
+    def _perform_login(self, username, password):
+        login_page = self._download_webpage(
+            'https://www.playsuisse.ch/api/sso/login', None, note='Downloading login page',
+            query={'x': 'x', 'locale': 'de', 'redirectUrl': 'https://www.playsuisse.ch/'})
+        settings = self._search_json(r'var\s+SETTINGS\s*=', login_page, 'settings', None)
```

Comment:
let's extract out the useful bits
```suggestion
        settings = self._search_json(r'var\s+SETTINGS\s*=', login_page, 'settings', None)

        csrf_token = settings['csrf']
        query = {'tx': settings['transId'], 'p': self._LOGIN_PATH}
```